### PR TITLE
Update aryamaan.json

### PR DESCRIPTION
**Pull request by HalalJadeja**
added mx and txt records

### Requirements
- [✅] You're not using Vercel or Netlify.
- [✅] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [✅] The website is reachable.
- [✅] The CNAME record doesn't contain `https://` or `/`.  
- [✅] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://aryamaan.is-a.dev

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

File: domains/aryamaan.json [hunk 1]
```diff
@@ -6,7 +6,12 @@
       },
 
       "record": {
-        "CNAME": "halaljadeja.github.io"
+        "CNAME": "halaljadeja.github.io",
+          "MX": [
+              "mx1.improvmx.com.",
```
Comment (WilliamDavidHarrison) [inline domains/aryamaan.json:11]: Please remove the `.` at the end of the line.

File: domains/aryamaan.json [hunk 2]
```diff
@@ -6,7 +6,12 @@
...
+              "mx2.improvmx.com."
```
Comment (WilliamDavidHarrison) [inline domains/aryamaan.json:12]: Please remove the `.` at the end of the line.

File: domains/notjadeja.json [hunk 3]
```diff
@@ -6,7 +6,12 @@
       },
 
       "record": {
-        "CNAME": "halaljadeja.github.io"
+        "CNAME": "halaljadeja.github.io",
+          "MX": [
+              "mx1.improvmx.com.",
+              "mx2.improvmx.com."
+              ],
+          "TXT:" "v=spf1 include:spf.improvmx.com ~all"
```
Comment (WilliamDavidHarrison) [inline domains/notjadeja.json:14]: Please change it to `"TXT":`

File: domains/aryamaan.json [hunk 4]
```diff
@@ -6,7 +6,12 @@
...
+              ],
+          "TXT:" "v=spf1 include:spf.improvmx.com ~all"
```
Comment (WilliamDavidHarrison) [inline domains/aryamaan.json:14]: Please change it to `"TXT":`

File: domains/notjadeja.json [hunk 5]
```diff
@@ -6,7 +6,12 @@
       },
 
       "record": {
-        "CNAME": "halaljadeja.github.io"
+        "CNAME": "halaljadeja.github.io",
+          "MX": [
+              "mx1.improvmx.com.",
```
Comment (WilliamDavidHarrison) [inline domains/notjadeja.json:11]: Please remove the `.`

File: domains/notjadeja.json [hunk 6]
```diff
@@ -6,7 +6,12 @@
...
+              "mx2.improvmx.com."
```
Comment (WilliamDavidHarrison) [inline domains/notjadeja.json:12]: Please remove the `.`

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
oops wait, seems like CNAME records can't be combined with MX records nor any other types of records, sorry :(((((

**Comment by HalalJadeja** [comment]
> then how shall i add mx and txt records?

**Comment by hackermondev** [comment]
@HalalJadeja You can only use MX and TXT records with A records.

File: domains/aryamaan.json [hunk 7]
```diff
@@ -6,7 +6,12 @@
       },
 
       "record": {
-        "CNAME": "halaljadeja.github.io"
+        "CNAME": "halaljadeja.github.io",
```
Comment (WilliamDavidHarrison) [inline domains/aryamaan.json:9]: You need to use an A record here.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: domains/aryamaan.json [hunk 8]
```diff
@@ -4,9 +4,19 @@
         "username": "HalalJadeja",
         "email": "9236.stkabirdio@gmail.com"
       },
-
       "record": {
-        "CNAME": "halaljadeja.github.io"
+        "CNAME": "halaljadeja.github.io",
```
Comment (WilliamDavidHarrison) [inline domains/aryamaan.json:8]: Please remove the CNAME.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**